asset/asset_test.go

```diff
@@ -198,18 +198,20 @@ func TestAssetEncoding(t *testing.T) {
 		require.True(t, a.DeepEqual(&b))
 	}
 
-	split := &Asset{
-		Version: 1,
-		Genesis: Genesis{
-			FirstPrevOut: wire.OutPoint{
-				Hash:  hashBytes1,
-				Index: 1,
-			},
-			Tag:         "asset",
-			MetaHash:    [MetaHashLen]byte{1, 2, 3},
-			OutputIndex: 1,
-			Type:        1,
+	splitGen := Genesis{
```

Comment:
nit (commit message): what does "serde" mean? Sounds like a vim command :)

Reply:
aaaah, okay, got it. how about "ser/de-ser"? Still pretty short but IMO a bit easier to decipher.

Reply:
I also thought it was some fancy unix command or something 😂